@uportal/dashboard-carousel/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "dashboard-carousel",
+  "version": "0.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "start": "vue-cli-service serve",
+    "build": "vue-cli-service build --target wc --name dashboard-carousel"
+  },
+  "dependencies": {
+    "@uportal/open-id-connect": "^1.19.1",
+    "jquery": "^3.3.1",
+    "ky": "^0.5.1",
+    "vue": "^2.5.17",
+    "vue-async-computed": "^3.5.0",
+    "vue-slick": "^1.1.15"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/preset-env": "^7.2.0",
+    "@vue/cli-plugin-babel": "^3.2.0",
+    "@vue/cli-service": "^3.2.0",
```

Comment:
```suggestion
    "@vue/cli-service": "^3.2.3",
```

Reply:
Security fixes through a transitive dependency, `webpack-dev-server`

Reply:
updated in e763879